airflow/www/static/js/dag/Main.tsx

```diff
@@ -62,6 +64,9 @@ const Main = () => {
     onStatusHover.cancel();
   };
 
+  const gridWidth = localStorage.getItem(`grid-width-${dagId}`) || undefined;
+  const saveWidth = debounce((w) => localStorage.setItem(`grid-width-${dagId}`, w));
```

Comment:
Default value for `debounce` is 0 ms (~ no debounce). We should specify a value:
```suggestion
  const saveWidth = debounce((w) => localStorage.setItem(`grid-width-${dagId}`, w), localStorageDelay);
```

I tried that locally, but this change is not enough => then the function is only triggered when we stop resizing. (not during resizing, which could be great if this was expected :joy: ) Also, maybe those functions should be defined outside of the component. I don't think they should change between renders, or should be bound to the component.